volttron/platform/certs.py

```diff
@@ -358,18 +359,72 @@ def ca_exists(self):
         """
         return os.path.exists(self.cert_file(self.root_ca_name))
 
+    def get_cert_subject(self, name):
```

Comment:
have this used a namedtuple rather than a dictionary.  Then the user can access it by instance.common_name rather than have to "know" it's common-name.

Reply:
Again something I looked at but I like your solution better though I just wanted the common name for the csr request

Reply:
Not using a named tuple as I want to return subject detail that map the yml configuration. Which would I would have one of the dict key as 'common-name'.